src/_guides/libraries/writing-package-pages.md

```diff
@@ -38,130 +38,152 @@ follow these links:
-
-1. Write a short description at the top of the README
-2. Include visual content to explain things
-3. Utilize lists to present key information
-4. Include at least one usage example
-5. Use Dart code formatting
-6. Mention terms that people might use to search for
-7. Tell users where they can go next
-
-Throughout this document, we’ll use the README of the package
-[`in_app_purchase`][] as a “full” example.
-Depending on the complexity of your package,
-you can choose to go as simple as the README of the [`yaml`][] package. 
+People who find your package on pub.dev are likely to
+quickly scan the README when deciding whether to try your package.
+A good README catches the reader's attention and
+shows that your package is worth trying.
 
 {{ site.alert.note }}
-  Your README will be used in multiple places and contexts,
-  such as API references, in addition to the package page on pub.dev. 
+  The package README is used in multiple ways.
+  For example, its content appears not only in the package page on pub.dev,
+  but also in [dartdoc][]-produced API reference documentation.
 {{ site.alert.end }}
 
+Although this page features the [`in_app_purchase`][] package README,
+yours might not need to be as large or detailed.
+If your package is simple and has no associated UI,
+its README might look more like the one for the [`yaml`][] package. 
+
+
 ## Seven tips for good README
 
-### 1. Write a short description at the top of the README
+Here are some suggestions for creating a README
+that works well on pub.dev:
+
+1. [Put a short description at the top](#tip1)
+2. [Include visual content](#tip2)
+3. [Use lists to present key information](#tip3)
+4. [Include usage examples](#tip4)
+5. [Use Dart code formatting](#tip5)
+6. [Mention related terms](#tip6)
+7. [Tell users where to go next](#tip7)
+
+
+### 1. Put a short description at the top {#tip1}
 
 According to our user research,
 package users spend only a few seconds to
-read the package description and make a decision whether to
-read the rest of the README or not.
+read the package description and decide whether to
+read the rest of the README.
 Thus, you should concisely describe what the package does or achieves,
 at a glance.
 Spend some time to craft a short and sweet description and
 help the user to make decisions.
-There’s no need to write the package name again on the top. 
 
-See examples of good descriptions below.
+{{site.alert.tip}}
+  Don't write the package name again at the top.
+  It's already visible in the pub.dev UI.
+{{site.alert.end}}
 
-*   A Flutter plugin for showing rainbows.
-*   This plugin allows Flutter apps to categorize bird sounds using machine learning.
+Here are some examples of good descriptions:
 
-If there is important information users should know about,
-such as project status or constraints, note that at the top too. 
+*   `A Flutter plugin for showing rainbows.`
+*   `Use machine learning to categorize bird sounds.`
```

Comment:
It's a plane...it's super man...it's a round hummingbird...it's dash!

Reply:
It's Mega-Dash! 😆